### DISCUSSION TITLE
Using a custom model created in birdnet analyser in a BirdNET-PI station

### DISCUSSION THREAD

**Discussion by gutocarv** [mcguirepr89/BirdNET-Pi#1156] [Q&A]
Hi all
We are working on monitoring the Cherry-throated-tanager (Nemosia rourei), a highly endangered species, in the forests of Southeast Brazil. We have experimentally installed a BirdNET-PI station, and the results have been very promising for numerous species. However, the current BirdNET model does not identify our target species, the Cherry-throated-tanager (Nemosia rourei).
So, we created a model using [BirdNET Analyzer](https://github.com/kahst/BirdNET-Analyzer) to use it in the BirdNETPI station, but we couldn't get it to work.

Question: Is it possible to use a model created in BirdNET Analyzer on BirdNETPI?
If yes, how can we do this? What parameters need to be adjusted?"

Our team is not familiar with model creation or the use of TensorFlow, but we managed to create a highly accurate model using BirdNET Analyzer. It would be amazing to be able to use it in the field to monitor the Cherry-throated Tanager.

Any help will be higly appreciated

**Comment by rdz-oss** [comment]
Hi @gutocarv  - I am linking my reply here as others maybe also are interested. It takes some work to make a server.py to server.py bridge between BirdNET-Pi and BirdNET-Analyzer but it has been done at least twice by now. My solution has not become a pull request because I am making a system for bats out of it (so more changes were necessary). But can be done so: https://github.com/rdz-oss/BattyBirdNET-Pi/discussions/1#discussioncomment-8804055

I was inspired by this solution, which did how ever not work anymore after changes made to BirdNET-Analyter: https://github.com/mmcc-xx/BirdCAGE/blob/main/BirdNET-Pi/server.py
